vignettes/IntroToAnnotationPackages.Rmd

```diff
@@ -0,0 +1,383 @@
+---
+title: "AnnotationDbi: Introduction To Bioconductor Annotation Packages"
+author: 
+  - name: "Marc Carlson"
+  - name: "Beryl Kanali"
+    affiliation: "Converted vignette from Sweave to Rmarkdown"
+date: "Last modified: November 2022; Compiled: `r format(Sys.time(), '%B %d , %Y')`"
+
+vignette: >
+  %\VignetteIndexEntry{1. Introduction To Bioconductor Annotation Packages}
+  %\VignetteDepends{org.Hs.eg.db, TxDb.Hsapiens.UCSC.hg19.knownGene, EnsDb.Hsapiens.v75}
+  %\VignetteKeywords{annotation, database}
+  %\VignettePackage{AnnotationDbi}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+```{r  label = fig:dbtypes, echo = FALSE, fig.align="center", fig.cap = "Annotation Packages: the big picture", out.width = '60%'}
+knitr::include_graphics("databaseTypes.png")
+```
```

Comment:
I think fig is not a valid chunk label, fig.align is by default center so we don't need to mention it here please correct this code chunk label this should be.

````
```{r dbtypes, fig.cap = "Annotation Packages: the big picture", echo = FALSE, fig.width=0.6, fig.cap = "Annotation Packages: the big picture"}
knitr::include_graphics("databaseTypes.png")
```
````